src/countdown/src/Countdown.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from 'vue'
 import type { ExtractPublicPropTypes } from '../../_utils'
 
-interface TimeInfo {
+export interface TimeInfo {
```

Comment:
Rename it to `CountdownTimeInfo` and export it from index of Countdown component.

Reply:
For exposed interface we need to give it a scope.

Reply:
That makes sense, thank you.